package.json

```diff
@@ -110,30 +110,27 @@
     "@shopify/sewing-kit": "^0.132.2",
     "@shopify/shrink-ray": "^2.3.1",
     "@shopify/splash": "^0.0.8",
-    "@storybook/addon-a11y": "^5.3.19",
-    "@storybook/addon-actions": "^5.3.19",
+    "@storybook/addon-a11y": "^6.0.12",
     "@storybook/addon-console": "^1.2.1",
     "@storybook/addon-contexts": "^5.3.19",
```

Comment:
Contexts is deprecated and doesn't have a v6 version, but we're stuck with it as the new args/toolbars approach of configuring the "Strict Mode" and "Design Language" toggle buttons doesn't support deeplinking. I've raised a bug to request support for that so we can move away: https://github.com/storybookjs/storybook/issues/11604